torch/_export/serde/serialize.py

```diff
@@ -1223,7 +1223,7 @@ def serialize_hoo_outputs(self, node: torch.fx.Node) -> List[Argument]:
         """
         meta_val = node.meta["val"]
 
-        if isinstance(meta_val, tuple):
+        if isinstance(meta_val, (list, tuple)):
```

Comment:
Is this a bugfix? If so, can it be triggered outside of this PR? If so, maybe split it out as a separate PR?

Reply:
Yeah, this should be a separate PR.